vcpkg/consume/snippets/versioning-cmakelists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Ditto would be nice to have an examples directory somewhere. No change requested here.